### ISSUE TITLE
Bundle transaction cannot resolve valueReference.reference

### ISSUE THREAD

**Issue by wetret** [highmed/highmed-dsf#31]
If posting a transaction bundle, the relative reference IDs of type valueReference.reference are not properly resolved and replaced by the newly generated server IDs.

Example:
If we have a ResearchStudy in a transaction Bundle like:
```
{
      "fullUrl": "urn:uuid:ae194285-d08e-46c7-a9a7-9a47624dd8aa",
      "resource": {
          "resourceType": "ResearchStudy",
           ...
      },
      "request": {
        "method": "POST",
        "url": "ResearchStudy"
      }
}
```
And a Task resource with an input like:
```
"input": [
  {
     "type": {
          "coding": [
               {
                   "system": "http://highmed.org/fhir/CodeSystem/bpmn-message",
                   "code": "research-study-reference"
               }
            ]
       },
       "valueReference": {
           "reference": "urn:uuid:ae194285-d08e-46c7-a9a7-9a47624dd8aa"
       }
   }
]
```
The value of `valueReference.reference` is not properly resolved by the id of the new generated ResearchStudy.

**Comment by hhund** [comment]
This bug might be a missing feature in the  org.highmed.dsf.fhir.service.ReferenceExtractorImpl.getReferences(Task resource) method. Currently the ReferenceExtractor does not look at input/output fields.

Please create a branch an add a test/test-class analog to org.highmed.dsf.fhir.integration.OrganizationIntegrationTest or org.highmed.dsf.fhir.integration.BinaryIntegrationTest

Feel free to create a failing Bundle in the new integration test case and use getWebserviceClient().postBundle(bundle) to post it to the integration-test server.

**Comment by wetret** [comment]
Implemented with 5f3a67a

**Comment by hhund** [comment]
merged with #34
